### PR TITLE
fix(product): The bin scripts should include a shebang and import in the body

### PR DESCRIPTION
**Pull request by adrien2p**

### CODE REVIEW

**Comment by adrien2p** [comment]
> lgtm, but haven't tested - did you?

Not yet, unfortunately never got the snap 😅

File: packages/product/src/scripts/migration-up.ts [hunk 1]
```diff
@@ -33,11 +33,15 @@ export async function runMigrations({
 
   try {
     const migrator = orm.getMigrator()
-    await migrator.up()
+    const pendingMigrations = await migrator.getPendingMigrations()
+    logger.info("pendingMigrations", pendingMigrations)
```
Comment (olivermrbl) [inline packages/product/src/scripts/migration-up.ts]:
```suggestion
```

File: packages/product/src/utils/create-connection.ts [hunk 2]
```diff
@@ -1,35 +1,26 @@
...
-      path: resolve(__dirname, "../../dist/migrations"),
-      pathTs: resolve(__dirname, "../../src/migrations"),
-      glob: "!(*.d).{js,ts}",
-      disableForeignKeys: false,
-      silent: false,
-      dropTables: false,
-      transactional: true,
-      allOrNothing: true,
-      safe: true,
-      generator: TSMigrationGenerator,
```
Comment (olivermrbl) [inline packages/product/src/utils/create-connection.ts:32]: question: We don't need these?

**Review by olivermrbl — CHANGES_REQUESTED** [review]

File: packages/product/src/utils/create-connection.ts (hunk 2 above)
Comment (adrien2p) [reply packages/product/src/utils/create-connection.ts:32]: No we don't as per mikro orm [doc](https://mikro-orm.io/docs/migrations#running-migrations-in-production)

File: packages/product/src/scripts/migration-up.ts (hunk 1 above)
Comment (adrien2p) [reply packages/product/src/scripts/migration-up.ts]: This is to show which migrations will be ran, do you think it is not needed?

Comment (olivermrbl) [reply packages/product/src/scripts/migration-up.ts]: In that case, I would do: `"Running pending migrations: "`

Comment (adrien2p) [reply packages/product/src/scripts/migration-up.ts]: That is ok for me 👍

**Comment by olivermrbl** [comment]
@adrien2p I think the `pg` dep in integration tests is no longer needed. Can you remove that before we merge?

**Comment by adrien2p** [comment]
I think it is needed si ce we install typeorm and we are using postgres no?

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**